### PR TITLE
Fixes performance regression in GO 1.6

### PR DESCRIPTION
**Pull request by darrenstahlmsft**
Signed-off-by: Darren Stahl <darst@microsoft.com>

### CODE REVIEW

**Comment by darrenstahlmsft** [comment]
@jstarks

File: file.go [hunk 1]
```diff
@@ -117,6 +118,8 @@ func (f *win32File) prepareIo() (*ioOperation, error) {
 
 // ioCompletionProcessor processes completed async IOs forever
 func ioCompletionProcessor(h syscall.Handle) {
+	// Set the timer resolution to 1. This fixes a performance regression in golang 1.6.
+	timeBeginPeriod(1)
```
Comment (jhowardmsft) [inline file.go]: Think this should be 1000 (?) @jstarks

Comment (darrenstahlmsft) [inline file.go]: Talked to @jstarks, he said it should be 1.

Comment (jhowardmsft) [inline file.go]: SGTM, and LGTM

**Comment by jstarks** [comment]
LGTM

**Comment by tmm1** [comment]
I believe this can be reverted for go 1.7 and 1.8

**Pull request merged**